app/.eslintrc.json

```diff
@@ -9,6 +9,9 @@
     "nava",
     "prettier"
   ],
+  "globals": {
+    "JSX": true
+  },
```

Comment:
You can remove this if you change the type in `src/components/Layout.tsx` from `JSX.Element` to `ReactElement`.